content/getting_started/CUDAGuide.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "CUDA Guide"
+date: "2021-01-20"
+draft: false
+weight: 5
+---
+
+## Reference C++ example
+
+Suppose we wanted to port the following C++ code to CUDA, with Enzyme autodiff support:
```

Comment:
Can you add a note here saying that support CUDA is still in progress research and thus experimental.